src/schema/v2/artwork/index.ts

```diff
@@ -107,6 +122,9 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
           return artistLoader(artist.id).catch(() => null)
         },
       },
+      insights: {
```

Comment:
**(suggestion):** Should we keep the naming consistent and name it `marketPriceInsights` or `priceInsights`?